src/main/java/ru/yandex/practicum/filmorate/storage/impl/FriendshipDaoImpl.java

```diff
@@ -0,0 +1,58 @@
+package ru.yandex.practicum.filmorate.storage.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+import ru.yandex.practicum.filmorate.model.User;
+import ru.yandex.practicum.filmorate.storage.FriendshipDao;
+
+import java.util.List;
+
+@Component
+public class FriendshipDaoImpl implements FriendshipDao {
+
+    JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public FriendshipDaoImpl(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public void create(int userId, int friendId) {
+        jdbcTemplate.update("INSERT INTO friendship (USER_ID, FRIEND_ID) VALUES (?, ?)",
+                userId, friendId);
+    }
+
+    @Override
+    public List<User> readFriendsByUserId(int userId) {
+        return jdbcTemplate.query("SELECT * " +
```

Comment:
SELECT * FROM users, friends WHERE users.user_id = friends.friend_id AND friends.friend_id = ?